usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1278,7 +1278,7 @@
                     "prime_award_obligated_amount_funded_by_COVID-19_supplementals",
                     None,
                 ),  # Annotation is used to create this column
-                ("prime_award_base_action_date", "broker_subaward__action_date"),
+                ("prime_award_base_action_date", "action_date"),
```

Comment:
Because I'm no longer sure. Is this definitely `action_date` or should it be `sub_action_date`?